import-dashboards.py

```diff
@@ -255,8 +255,19 @@ def rename_panels():
             if os.path.exists(panel_path):
                 with open(panel_path, 'r') as f:
                     panel_params = json.loads(f.read())
+                    if panel == panel_params['id']:
+                        print 'skipped'
+                        continue
                     print '%r' % (panel_params['id'],)
+                    if os.path.isdir(os.path.join(GRAFANA_PLUGINS_DR, panel_params['id'])):
+                        try:
+                            shutil.rmtree(os.path.join(GRAFANA_PLUGINS_DR, panel_params['id']))
+                        except Exception as err:
+                            print '  * Failed to remove %s: %s' % (os.path.join(GRAFANA_PLUGINS_DR, panel_params['id']), err)
+                            continue
                     os.rename(os.path.join(GRAFANA_PLUGINS_DR, panel), os.path.join(GRAFANA_PLUGINS_DR, panel_params['id']))
+            else:
+                print 'skipped'
```

Comment:
```suggestion
                print('Path %r does not exist - skipped' % (panel_path,))
```
Too far from `if` - maybe this msg or please add a comment.

Reply:
I have added more details into the output message.